docs/developer-docs/defi/tokens/create.mdx

```diff
@@ -0,0 +1,177 @@
+---
+keywords: [intermediate, defi, overview, tokens, deploy a token, create a token, icrc1]
+---
+
+import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
+
+# Create a token
+
+<MarkdownChipRow labels={["Intermediate"]} />
+
+## Overview
+
+Custom tokens can be created and deployed on ICP using the [ICRC-1](https://github.com/dfinity/ICRC-1/tree/main/standards/ICRC-1) or [ICRC-2](https://github.com/dfinity/ICRC-1/tree/main/standards/ICRC-2) token standards.
+
+The ICRC-1 standard is considered the 'base' standard, as it defines a set of general functionalities that all ledgers must adhere to. Any tokens and their corresponding ledgers must fulfill all requirements of the ICRC-1 standard. Read about [token standards](token-standards.mdx) to learn more about what token standards are and the functionalities each one supports.
+
+This guide will provide a quickstart on how you can create and deploy your own custom token using the ICRC-1 standard. 
+
+## Setup a local ICRC-1 ledger
+
+To create a new ICRC-1 token, you will need to first download and configure a local project for the ICRC-1 ledger canister. This is because each token will need its own ledger canister to record the token's balances and transactions. [Follow the ICRC-1 ledger documentation for instructions](/docs/current/developer-docs/defi/icrc-1/icrc1-ledger-setup).
```

Comment:
Did we not just delete this reference to icrc1-ledger-setup?

Reply:
This PR moves the steps of the ledger setup that create a new token into this new 'Create' page. The ledger setup doc now contains just the steps to obtain the Wasm/Candid files, then points to this 'Create' page to setup the variables to create a new token and initialize the ledger for that token.